cmd/vclusterctl/cmd/platform/start.go

```diff
@@ -83,6 +85,15 @@ func (cmd *StartCmd) Run(ctx context.Context) error {
 			cmd.Version = latestVersion
 		}
 	}
+
+	// if < v4.0.0 then use ChartName loft
+	parsedVersion, err := semver.Parse(strings.TrimPrefix(cmd.Version, "v"))
+	if err != nil {
+		return fmt.Errorf("parse provided version %s: %w", cmd.Version, err)
+	} else if parsedVersion.LT(semver.MustParse("4.0.0-alpha.0")) && cmd.ChartName == "vcluster-platform" {
```

Comment:
Why is this hard-coded? 

Upgraded platforms will still have `loft`, but only fresh installs will have `vcluster-platform`. 

Also, users could choose any namespace name that they want.

Reply:
This is not about the namespace its the chart that doesn't exist prior to v4.0.0-alpha.0